chaos_wrapper.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+function rollback() {
+    git checkout master
+    git checkout HEAD^
```

Comment:
So now the question is should we do it always if there is an abnormal exit or only if it crashes in the first 60s, as you suggested [above](https://github.com/chaosbot/Chaos/pull/318#discussion_r118827775)...